docs/developers/cosmwasm-dependency.md

```diff
@@ -78,3 +78,9 @@ go mod tidy -compat=1.17
 go mod download
 make install
 ```
+
+## celestia-node
```

Comment:
Let's make this capitalized, so "Celestia Node"

Reply:
✅